sdl_ios/SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -295,16 +311,15 @@ - (void)openSession {
         } else {
             if ([self.protocolString isEqualToString:CONTROL_PROTOCOL_STRING]) {
```

Comment:
Thanks.
The SDLDebugTool also makes data visible to us VIA Siphon.  We need this information since multiapp/background connection is so finicky we have to have these debug lines to ensure unexpected behavior is normal.